Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml

```diff
@@ -1,3 +1,21 @@
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitDeadSquad
+  name: Dead Squad uniform
```

Comment:
Should be `deathsquad`, entity ID too

Reply:
its deathsquad not Dead Squad

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/128301315/2fa30a03-728b-43a5-99a3-9d088f7a3883)

Reply:
so fix the typo pro dont continue it
also you ignore ClothingOuterHardsuitDeathsquad
and the comment above it